c2corg_ui/views/document.py

```diff
@@ -366,9 +366,9 @@ def render_page(content1, content2):
                 'version1': version1,
                 'version2': version2,
                 'geometry1': doc_v1['geometry']['geom']
-                if doc_v1['geometry'] else None,
+                if 'geometry' in doc_v1 else None,
```

Comment:
Thanks. API was ok, it works now. I was just missing new col. (is_profile_public) in Users table, so some documents couldn't be selected.